### DISCUSSION TITLE
Inline queries? Trigger not fired...

### DISCUSSION THREAD

**Discussion by whilefree** [westacks/telebot#108] [Q&A] [answered]
Hi. I'm new to this thing and this may be a pretty noobish question. But here is my code:

```php
<?php

namespace App\Http\UpdateHandlers;

use WeStacks\TeleBot\Handlers\UpdateHandler;
use Illuminate\Support\Facades\Log;

class InlineTest extends UpdateHandler
{
    public function trigger(): bool
    {
        Log::info('Test trigger called');
        return isset($this->update->inline_query);
    }

    public function handle()
    {
        // Log the update for debugging purposes
        Log::info('Test handle called');
    }
}
```

Using laravel, and polling (not webhooks) The trigger function is only fired when a message is sent to the bot itself. When using inline messaging, the trigger function is not even fired to check if it's an inline_query or what. This line:

return isset($this->update->inline_query);

Is based on chatgpt which seems to be incorrect. In actual fact chatgpt says a lot of nonsense when it comes to telegram bots, which I guess is due to lack of online tutorials.

Please guide me in the right direction on how to handle inline messages and callback queries, etc.

Thanks.

**Comment by punyflash** [comment, marked as answer]
Probably you missed something from [API docs](https://core.telegram.org/bots/api#inline-mode.) or forgot to turn on [inline mode](https://core.telegram.org/bots/inline). There is nothing wrong with a code
